packages/terra-popup/src/_PopupContent.jsx

```diff
@@ -82,6 +82,10 @@ const propTypes = {
    * The function returning the frame html reference.
    */
   refCallback: PropTypes.func,
+  /*
+   * String that labels the popup for screen readers.
+   */
+  ariaLabel: PropTypes.string,
```

Comment:
this changes are not required !! since we are alreading passing aria-label to popup-content on https://github.com/cerner/terra-framework/pull/1779/files#diff-e35d8cfdc3251ddabe24a68a9421968d4d86a1a2b0c76242ee861906306cd939

Reply:
Done.